src/components/Link.tsx

```diff
@@ -97,7 +97,7 @@ const Link: React.FC<IProps> = ({
 
   const isExternal = to.includes("http") || to.includes("mailto:")
   const isHash = isHashLink(to)
-  const isGlossary = to.includes("glossary#")
+  const isGlossary = to.includes("glossary" && "#")
```

Comment:
`String.includes` only accepts 1 argument so this will ignore "glossary" match any link including "#".

```suggestion
  const isGlossary = to.includes("glossary") && to.includes("#")
```